src/app/vault/vault.component.ts

```diff
@@ -159,6 +163,7 @@ export class VaultComponent implements OnInit, OnDestroy {
                         if (this.cipherId != null && uCipher != null && uCipher.id === this.cipherId &&
                             uCipher.login != null && uCipher.login.username != null) {
                             this.copyValue(uCipher.login.username, 'username');
+                            this.minimizeIfNeeded();
```

Comment:
So this feature should only work when using the keyboard shortcuts? What about when copying username/password to clipboard from other ways, like the view page or the context menu?

Reply:
Implemented with 8849399. Part of this is implemented in [PR #74](https://github.com/bitwarden/jslib/pull/74).